proof/invariant-abstract/ARM/ArchEmptyFail_AI.thy

```diff
@@ -37,7 +37,7 @@ context Arch begin global_naming ARM
 crunch (empty_fail) empty_fail[wp, EmptyFail_AI_assms]: possible_switch_to,set_thread_state_act
   (simp: kernel_object.splits option.splits arch_cap.splits cap.splits endpoint.splits
          bool.splits list.splits thread_state.splits split_def catch_def sum.splits
-         Let_def wp: zipWithM_x_empty_fail)
+         Let_def wp: empty_fail_zipWithM_x)
```

Comment:
I suspect you can simply remove the lemma from `wp: ..`, because it should already be declared `[wp]`, but not worth another re-run.